Explorer/Assets/DCL/MapRenderer/MapLayers/Atlas/ParcelAtlas/ParcelChunkAtlasController.cs

```diff
@@ -0,0 +1,97 @@
+using Cysharp.Threading.Tasks;
+using DCL.MapRenderer.CoordsUtils;
+using DCL.MapRenderer.Culling;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.MapRenderer.MapLayers.Atlas
+{
+    internal class ParcelChunkAtlasController : MapLayerControllerBase, IAtlasController
+    {
+        public delegate UniTask<IChunkController> ChunkBuilder(Vector3 chunkLocalPosition, Vector2Int coordsCenter, Transform parent, CancellationToken ct);
+
+        public const int CHUNKS_CREATED_PER_BATCH = 5;
+
+        private readonly int chunkSize;
+        private readonly int parcelsInsideChunk;
+        private readonly ChunkBuilder chunkBuilder;
+        private readonly List<IChunkController> chunks;
+
+        private int parcelSize => coordsUtils.ParcelSize;
+
+        public ParcelChunkAtlasController(Transform parent, int chunkSize,
+            ICoordsUtils coordsUtils, IMapCullingController cullingController, ChunkBuilder chunkBuilder)
+            : base(parent, coordsUtils, cullingController)
+        {
+            this.chunkSize = chunkSize;
```

Comment:
do you need to store chunk size? I see all methods that use it get it as a parameter 🤔